templates/default/custom.properties.erb

```diff
@@ -127,7 +127,7 @@ org.opencastproject.db.jdbc.driver=com.mysql.jdbc.Driver
 #  .url=jdbc:h2:${org.opencastproject.storage.dir}/db
 #  .user=sa
 #  .pass=sa
-org.opencastproject.db.jdbc.url=jdbc:mysql://<%= @database[:host] %>/<%= @database[:database] %>
+org.opencastproject.db.jdbc.url=jdbc:mysql://<%= @database[:host] %>/<%= @database[:database] %>?autoReconnect=true&useSSL=false
```

Comment:
I'm curious why you added the useSSL=false here. This will impact the TSLv

Reply:
I tried several suggestions I found from googling with no success and finally decided since the RDS cluster is running on a private subnet it was OK to just disable SSL. I think this happened **AFTER** i'd already introduced the alternative `java.security` config file, so I'm not 100% sure if the latter is still necessary. 

I'm going to go ahead and revisit this and try some additional things to make sure this is the best solution. For instance, that SO thread mentions some debugging options I didn't try. Also I guess in some cases the tls connection string parameter is `tlsVersion` instead of `enabledTLSProtocols`.

Reply:
I tried again with the same result. I was able to determine that with `useSSL=false` we no longer need to provide a modified `java.security` file. So that's saving a bit of complexity.